cmd/node/node.go

```diff
@@ -306,6 +297,20 @@ func (app *App) introduction() {
 
 // Initialize sets up an exit signal, logging and checks the clock, returns error if clock is not in sync.
 func (app *App) Initialize() (err error) {
+	// tortoise wait zdist layers for hare to timeout for a layer. once hare timeout, tortoise will
+	// vote against all blocks in that layer. so it's important to make sure zdist takes longer than
+	// hare's max time duration to run consensus for a layer
+	maxHareLayerDurationSec := app.Config.HARE.WakeupDelta + app.Config.HARE.LimitIterations*4*app.Config.HARE.RoundDuration
```

Comment:
maybe 4 should be defined as a constant in hare module?

Reply:
added. thanks